script/reference-device/dhcp6_pd_daemon.py

```diff
@@ -107,8 +107,17 @@ def connect_to_signal():
 
 
 def check_and_reconnect(dbus_obj):
+    global bus
     if dbus_obj is None:
         connect_to_signal()
+    bus.watch_name_owner(bus_name='io.openthread.BorderRouter.wpan0',
```

Comment:
For this [watch_name_owner API](https://dbus.freedesktop.org/doc/dbus-python/dbus.bus.html?highlight=watch_name_owner#dbus.bus.BusConnection.watch_name_owner), what's the behavior of calling it multiple times?